### PR TITLE
Add the invite link to the slack 👼

### PR DESCRIPTION
**Pull request by vdemeester**
Let's hope it works for ever, otherwise we'll find a new solution.

/cc @bobcatfish @abayer 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: contact.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Some important channels:
 * `#admin` - Request slack related changes, e.g. proposing new channels
 * `#plumbing` - Request [infrastructure support](https://github.com/tektoncd/plumbing#support)
 
+You can join the `tektoncd` slack using the following link : [here](https://knative.slack.com/archives/CDCQ72D0R/p1556040805053000).
```
Comment (chmouel) [inline contact.md]: Humm did you mean the tekton slack ?

Comment (vdemeester) [reply contact.md]: oups =) wrong link :man_facepalming:

**Comment by chmouel** [comment]
Looks good altought reading the documentation here : https://get.slack.help/hc/en-us/articles/201330256-Invite-new-members-to-your-workspace

there is a note : 

```
Note: Invite links can be used by up to 2,000 people. You'll need to create a new link if you reach this limit.
```
Which I guess we need to keep in mind for the future

 (when we take over the world that's it!)

**Comment by bobcatfish** [comment]
Thank you @chmouel !!! :heart: :heart::heart:

**Comment by bobcatfish** [comment]
> Thank you @chmouel !!!

... and @vdemeester ... I CAN TELL WHO THE AUTHOR OF THE CL IS I SWEAR... :sweat_smile:

**Pull request merged**